Source/HttpMultipartParser/MultipartFormDataParser.cs

```diff
@@ -557,10 +557,11 @@ private void ParseStream(Stream stream, string boundary, Encoding encoding, int
             var streamingParser = new StreamingMultipartFormDataParser(stream, boundary, encoding, binaryBufferSize);
             streamingParser.ParameterHandler += parameterPart => Parameters.Add(parameterPart);
 
-            streamingParser.FileHandler += (name, fileName, type, disposition, buffer, bytes) =>
+            streamingParser.FileHandler += (name, fileName, type, disposition, buffer, bytes, partNo) =>

```

Comment:
I would rename in order to be consistent

```suggestion
            streamingParser.FileHandler += (name, fileName, type, disposition, buffer, bytes, partNumber) =>
```